provider/messagebird/messagebird.go

```diff
@@ -26,13 +31,26 @@ func NewMessageBird(config MessageBirdConfig) *MessageBird {
 	}
 	return &MessageBird{
 		client: client,
-		params: messagebird.MessageParams{
+		messageParams: messagebird.MessageParams{
 			Gateway: config.Gateway,
 		},
+		voiceMessageParams: messagebird.VoiceMessageParams{
+			Language: config.Language,
+			Voice: config.Voice,
+			Repeat: config.Repeat,
+		},
 	}
 }
 
 func (mb *MessageBird) Send(message sachet.Message) error {
-	_, err := mb.client.NewMessage(message.From, message.To, message.Text, &mb.params)
+	var err error=nil
+	switch message.Type {
+	case "text":
```

Comment:
```suggestion
	case "","text":
```
If you include the empty string here then it doesn't need to default to anything.

Reply:
Updated in e35c3d5383635563fa320caf098420c3b080e75c